scipy/spatial/transform/_rotation.pyx

```diff
@@ -1475,8 +1475,8 @@ cdef class Rotation:
         if len(seq) != 3:
             raise ValueError("Expected 3 axes, got {}.".format(seq))
 
-        intrinsic = (re.match(r'^[XYZ]{1,3}$', seq) is not None)
-        extrinsic = (re.match(r'^[xyz]{1,3}$', seq) is not None)
+        intrinsic = re.match(r'^[XYZ]{1,3}$', seq)
+        extrinsic = re.match(r'^[xyz]{1,3}$', seq)
```

Comment:
Note that we can't use `re.I` to condense these expressions into one, because there are Cython calls that depend on the individual cased matches (that repetition is what originally drew my attention here).